### PR TITLE
Fix issue 121

### PR DESCRIPTION
**Pull request by intendo**
Added field_name check to Atom.
Removed Tuple and atoms property from AtomList.

Fixed flake8 warnings.

### CODE REVIEW

File: apbs/geometry/sphere.py [hunk 1]
```diff
@@ -13,7 +13,7 @@ def spherical_distribution(self, npoints: int) -> Surface:
 
         Shamelessly copied over from the vacc routine.
```
Comment (sobolevnrm) [inline apbs/geometry/sphere.py:14]: We should consider replacing this with https://github.com/Electrostatics/osmolytes/blob/f8ac99827ba199b359ccf141b604a2b7362f9dbf/osmolytes/sasa.py#L31

**Review by sobolevnrm — APPROVED** [review]
See comment about eventually replacing sphere routine with https://github.com/Electrostatics/osmolytes/blob/f8ac99827ba199b359ccf141b604a2b7362f9dbf/osmolytes/sasa.py#L31

File: apbs/geometry/sphere.py (hunk 1 above)
Comment (intendo) [reply apbs/geometry/sphere.py:14]: Would we just add the osmolytes package and remove many of the geometry related functions?

**Review by aaronrphillips — APPROVED** [review]

**Comment by sobolevnrm** [comment]
Maybe eventually; however, there's a lot of functionality in that package right now that APBS wouldn't need immediately.

________________________________

**Pull request merged**